src/fundus/publishers/de/dw.py

```diff
@@ -13,11 +13,14 @@
     generic_date_parsing,
     generic_text_extraction_with_css,
     generic_topic_parsing,
+    strip_nodes_to_text,
 )
 
 
 class DWParser(ParserProxy):
     class V2(BaseParser):
+        VALID_UNTIL = datetime.date(2024, 1, 18)
+
         _paragraph_selector = CSSSelector("div.rich-text > p")
```

Comment:
DW seems to sometimes have additional tags at the end of the article (https://www.dw.com/de/trump-in-verleumdungsprozess-gegen-e-jean-carroll-zu-83-millionen-dollar-schadenersatz-verurteilt/a-68100499, https://www.dw.com/de/us-regierung-genehmigt-verkauf-von-f-16-kampfjets-an-türkei/a-68100064), unfortunately I didn't see an easy fix at the first glance, since they aren't any special tags and not always there.